.travis.yml

```diff
@@ -23,8 +23,10 @@ env:
 
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
-  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="account_invoice_line_default_account"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="account_invoice_line_sequence"
+  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1" EXCLUDE="account_invoice_line_default_account"
+  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1" EXCLUDE="account_invoice_line_sequence"
```

Comment:
Can runboat be informed about this?

https://runboat.odoo-community.org/api/v1/builds/bf7e8fb88-cda7-4c02-a0c2-5717acb6d8bf/init-log

Reply:
Runboat currently does not support conflicting addons, see https://github.com/sbidoul/runboat/issues/36